spring-cloud-stream/src/test/java/org/springframework/cloud/stream/config/ChannelBindingAdapterConfigurationTests.java

```diff
@@ -82,6 +82,7 @@ private void refresh() {
 			channel.setTapped(true);
 		}
 		this.adapter.setOutputChannels(channels);
+		this.adapter.stop();
```

Comment:
Actually, it doesn't matter here. All we need is to let the binding happens (when we restart the adapter) so that the appropriate binding names are set for the output channel.